core/validatorapi/validatorapi.go

```diff
@@ -483,6 +591,29 @@ func (c Component) verifyBlockSignature(ctx context.Context, block *spec.Version
 	return c.verifyParSig(ctx, core.DutyProposer, epoch, pubkey, sigRoot, sig)
 }
 
+func (c Component) verifyBlindedBlockSignature(ctx context.Context, block *eth2api.VersionedSignedBlindedBeaconBlock, pubkey core.PubKey, slot eth2p0.Slot) error {
+	epoch, err := c.epochFromSlot(ctx, slot)
+	if err != nil {
+		return err
+	}
+
+	var sig eth2p0.BLSSignature
+	if block.Version == spec.DataVersionBellatrix {
+		if block.Bellatrix.Signature == sig {
+			return errors.New("no bellatrix signature")
+		}
+		sig = block.Bellatrix.Signature
+	}
```

Comment:
should probably add a default error case (`verifyBlockSignature` also needs one).